packages/gravity-ui-nunjucks/__tests__/_helpers.js

```diff
@@ -0,0 +1,31 @@
+/**
+   * Takes an absolute path and returns an array
+   * of file names in a recursive way.
+   *
+   * @param  {string} startPath absolute path.
+   *
+   * @param {array} excludedFiles array with the name of the files
+   * to be excluded from the search
+   *
+   * @param {string} extension file extension type
+   *
+   * @return {array} List of files that match the extension given.
+   */
+const getComponentsNames = (components, excludedFiles) => (
+  components.filter((name) => !excludedFiles.includes(name))
+);
+
+/**
+   * Takes a set of objects and returns an array of formatted error messages.
+   *
+   * @param  {array} violations Set of violation objects.
+   *
+   * @return {array} Array of formatted messages.
+   */
+const getViolations = (violations) => violations
+  .map((violation) => `${violation.description}\n ${violation.help}\n Help: ${violation.helpUrl}\n`);
```

Comment:
Maybe update this to also include details about which element is causing the violation.

Reply:
Elements causing a11y violations are now included in the tests results 😄